src/SqlPersistence/SqlPersistenceStorageSessionExtensions.cs

```diff
@@ -15,12 +18,36 @@ public static class SqlPersistenceStorageSessionExtensions
         /// </summary>
         public static ISqlStorageSession SqlPersistenceSession(this SynchronizedStorageSession session)
         {
+            return GetSqlStorageSession(session);
+        }
+
+        internal static StorageSession GetSqlStorageSession(this SynchronizedStorageSession session)
```

Comment:
I've copied it over from NHIbernate. I agree it is not particularly helpful. User's don't configure it. You would get it if you would reference two persistence mechanisms and used, say, NH for the endpoint but in a handler you would use Sql persister extension method. Pretty unlikely.